<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSetting</class>
 <widget class="QDialog" name="ProjectSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" rowspan="3">
    <widget class="QTreeWidget" name="settingsList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>General</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Build</string>
      </property>
      <item>
       <property name="text">
        <string>GÃ©neral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Libs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Directories</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BuildManager</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Environment</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Build Configuration</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Run Configuration</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dependencies</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>General</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QStackedWidget" name="settingsPages">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_9">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>29</y>
         <width>921</width>
         <height>100</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Use custom Makefile for this project</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QLineEdit" name="makeFileEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>25</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="iconLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="iconEdit">
       <property name="geometry">
        <rect>
         <x>27</x>
         <y>1</y>
         <width>223</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="getIconButton">
       <property name="geometry">
        <rect>
         <x>256</x>
         <y>2</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="iconButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>0</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="build">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Active Configuration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="configsBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>BuildManager</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="compiler">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>442</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>label</zorder>
         <zorder>horizontalSpacer</zorder>
         <zorder>configsBox</zorder>
         <zorder>label_5</zorder>
         <zorder>compiler</zorder>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="buildPages">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="globalsPage">
          <layout class="QFormLayout" name="formLayout">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="targetLabel">
             <property name="text">
              <string>Target</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="targetEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="DestDirLabel">
             <property name="text">
              <string>Dest dir</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="destDirEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="ObjectsOutputLabel">
             <property name="text">
              <string>Objects output directory</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="objectsDirEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="typeLabel">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="templateBox">
             <item>
              <property name="text">
               <string>Console application</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Static Library</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dynamic Library</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
          <zorder>destDirEdit</zorder>
          <zorder>typeLabel</zorder>
          <zorder>DestDirLabel</zorder>
          <zorder>targetLabel</zorder>
          <zorder>ObjectsOutputLabel</zorder>
          <zorder>targetEdit</zorder>
          <zorder>templateBox</zorder>
          <zorder>objectsDirEdit</zorder>
         </widget>
         <widget class="QWidget" name="libsPage">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="linkerLayout"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="searchPathsPage">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="searchPathsLayout"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="CompilerPage">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="compilerOptionsLayout"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="EnvironmentPage">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="customVarsLayout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="config">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Active configuration :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="editConfigsBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addConfigButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renameConfigButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Rename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeConfigButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Build</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Clean</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Shadow build :</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Build directory :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="runConfig">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Executable</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="runExecutable">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="toolButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Arguments</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="runArgs">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Working directory</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="workingDirectory">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="toolButton_4">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="runInTerminalBox">
            <property name="text">
             <string>Run in terminal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Run configuration :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="runConfigsBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addRunConfigButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renameRunConfigButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Rename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeRunConfigButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dependencies">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QListWidget" name="dependenciesList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectSetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
